CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 - Use unstable libc++ ABI (#294).
 
+## [18.0.0]
+
+### Added
+
+- Documentation for the newlib overlay package (#399).
+- Libraries with exceptions to package (#351).
+- Enable cxx exceptions and rtti (#339).
```

Comment:
- Support for C++ exceptions and RTTI added in the standard libraries (https://github.com/ARM-software/LLVM-embedded-toolchain-for-Arm/pull/339).